packages/dapi-grpc/protos/platform/v0/platform.proto

```diff
@@ -7,36 +7,47 @@ package org.dash.platform.dapi.v0;
 
 service Platform {
-  rpc broadcastStateTransition (BroadcastStateTransitionRequest) returns (BroadcastStateTransitionResponse);
-  rpc getIdentity (GetIdentityRequest) returns (GetIdentityResponse);
-  rpc getIdentities (GetIdentitiesRequest) returns (GetIdentitiesResponse);
-  rpc getIdentityKeys (GetIdentityKeysRequest) returns (GetIdentityKeysResponse);
-//  rpc getIdentitiesKeys (GetIdentitiesKeysRequest) returns (GetIdentitiesKeysResponse);
-  rpc getIdentityBalance(GetIdentityRequest) returns (GetIdentityBalanceResponse);
+  rpc broadcastStateTransition(BroadcastStateTransitionRequest)
+      returns (BroadcastStateTransitionResponse);
+  rpc getIdentity(GetIdentityRequest) returns (GetIdentityResponse);
+  rpc getIdentities(GetIdentitiesRequest) returns (GetIdentitiesResponse);
+  rpc getIdentityKeys(GetIdentityKeysRequest) returns (GetIdentityKeysResponse);
+  //  rpc getIdentitiesKeys (GetIdentitiesKeysRequest) returns
+  //  (GetIdentitiesKeysResponse);
+  rpc getIdentityBalance(GetIdentityRequest)
+      returns (GetIdentityBalanceResponse);
   rpc getIdentityBalanceAndRevision(GetIdentityRequest)
-          returns (GetIdentityBalanceAndRevisionResponse);
-  rpc getProofs (GetProofsRequest) returns (GetProofsResponse);
-  rpc getDataContract (GetDataContractRequest) returns (GetDataContractResponse);
-  rpc getDataContracts (GetDataContractsRequest) returns (GetDataContractsResponse);
-  rpc getDocuments (GetDocumentsRequest) returns (GetDocumentsResponse);
-  rpc getIdentitiesByPublicKeyHashes (GetIdentitiesByPublicKeyHashesRequest) returns (GetIdentitiesByPublicKeyHashesResponse);
-  rpc getIdentityByPublicKeyHashes (GetIdentityByPublicKeyHashesRequest) returns (GetIdentityByPublicKeyHashesResponse);
-  rpc waitForStateTransitionResult (WaitForStateTransitionResultRequest) returns (WaitForStateTransitionResultResponse);
-  rpc getConsensusParams (GetConsensusParamsRequest) returns (GetConsensusParamsResponse);
+      returns (GetIdentityBalanceAndRevisionResponse);
+  rpc getProofs(GetProofsRequest) returns (GetProofsResponse);
+  rpc getDataContract(GetDataContractRequest) returns (GetDataContractResponse);
+  rpc getDataContracts(GetDataContractsRequest)
+      returns (GetDataContractsResponse);
+  rpc getDocuments(GetDocumentsRequest) returns (GetDocumentsResponse);
+  rpc getIdentitiesByPublicKeyHashes(GetIdentitiesByPublicKeyHashesRequest)
+      returns (GetIdentitiesByPublicKeyHashesResponse);
+  rpc getIdentityByPublicKeyHashes(GetIdentityByPublicKeyHashesRequest)
+      returns (GetIdentityByPublicKeyHashesResponse);
+  rpc waitForStateTransitionResult(WaitForStateTransitionResultRequest)
+      returns (WaitForStateTransitionResultResponse);
+  rpc getConsensusParams(GetConsensusParamsRequest)
+      returns (GetConsensusParamsResponse);
 }
 
 message Proof {
   bytes grovedb_proof = 1;
   bytes quorum_hash = 2;
   bytes signature = 3;
   uint32 round = 4;
+  bytes block_id_hash = 5;
```

Comment:
I just wondering if we could put this values into one hash which will be a part of state ID? Could you please remind me why do we have round here?

Reply:
Yeah, I understand. I just wondering if we could reduce the response size if we combine all data what user don't need on client side into one hash.